daemon/daemon.go

```diff
@@ -33,14 +33,22 @@ import (
 
 // enum to track current system state
 const (
-	unknown = iota
-	managed
-	operational
+	STARTING = 0 + iota
+	MANAGING
+	OPERATING
```

Comment:
Just FYI: It is fine for me (actually I did the same in the server package) having constants in upper case, as in C, but seems that go uses to name them as regular vars style. In any case, there is not strict naming convention for const (actually, if tests are passing that means it is valid)